sdk/storage/azure-storage-blobs/samples/blob_getting_started.cpp

```diff
@@ -5,19 +5,17 @@
 
 #include <azure/storage/blobs.hpp>
 
-#include "samples_common.hpp"
-
-SAMPLE(BlobsGettingStarted, BlobsGettingStarted)
-void BlobsGettingStarted()
+int main()
 {
   using namespace Azure::Storage::Blobs;
 
-  std::string containerName = "sample-container";
-  std::string blobName = "sample-blob";
-  std::string blobContent = "Hello Azure!";
+  const std::string connectionString = "";
+  const std::string containerName = "sample-container";
+  const std::string blobName = "sample-blob";
+  const std::string blobContent = "Hello Azure!";
 
   auto containerClient
-      = BlobContainerClient::CreateFromConnectionString(GetConnectionString(), containerName);
+      = BlobContainerClient::CreateFromConnectionString(connectionString, containerName);
```

Comment:
please keep the logic for reading the connection string from env-var.  We need it to be able to run the samples during CI gates.
And it is also ideal and consistent for customers ( avoid recompile for switching accounts )